### DISCUSSION TITLE
System.Net.Http error

### DISCUSSION THREAD

**Discussion by DFlowD31** [henrikfroehling/Trakt.NET#271] [Help] [answered]
Hello, I am new to this API and wanted to create my own app.
I have configured the client ID and Client Secret, but the TaskClient is giving me System.Net.Http 4.2.0.0 not found, even though I have 4.3.4 installed.

What would I be missing here?

Thanks & Regards,
dash
![Screenshot 2022-05-21 222101](https://user-images.githubusercontent.com/40911296/169653476-169fc433-6c00-48d3-a1db-bca5d5023bc5.png)

**Comment by henrikfroehling** [comment, marked as answer]
Hello,

I think this is not a problem with the library itself.
I'm not 100 % sure, but my guess would be that there are some mismatches regarding binding redirections.
So, I'm not an expert on that field, but which .NET version and which Visual Studio version do you use?
There might be a setting regarding binding redirections in your project properties.
I did a google search on this topic and there are some suggestions that Visual Studio might mix something up, which could lead to this error.

I think, the key hint would be "binding redirections".
